KosherCocoa/Library/Headers/KCJewishHolidays.h

```diff
@@ -84,7 +84,9 @@ enum kYomimTovim {
     kYomHashoah,
     kYomHazikaron,
     kYomHaatzmaut,
-    kYomYerushalayim
+    kYomYerushalayim,
+    kLagBaomer,
```

Comment:
I would do it, but I am pretty sure it would not be so simple. The index returned by KosherJava is 33, so I put it as the 33rd entry. I am not entirely sure how it affects other classes.